bench-show.cabal

```diff
@@ -96,13 +96,17 @@ source-repository head
     type: git
     location: https://github.com/composewell/bench-show
 
+flag charting
+  description: Build the modules that provide charting functionality
+  manual: True
+  default: False
```

Comment:
This should be true by default to keep it compatible with previous versions. We can name the flag no-charts to indicate that charts are on by default.